ide/utils/sdk/sdk_scripts.py

```diff
@@ -298,9 +298,9 @@ def build(ctx):
 
     ctx.set_group('bundle')
     ctx.pbl_bundle(binaries=binaries, js=ctx.path.ant_glob(['src/pkjs/**/*.js', 'src/pkjs/**/*.json']), js_entry_file='src/pkjs/{{pkjs_entry}}')
-"""
+""".replace('{{pkjs_entry}}', project.pkjs_entry_point)
```

Comment:
To clarify why this should always work:

- `pkjs_entry_point` is not-None if the project is a package, rocky, or [native project with commonjs selected]
- This function is only called for native projects, and the `{{pkjs_entry}}` replacement only happens with commonjs selected, so pkjs_entry_point will never be None.